pkg/rpcclient/rpc_test.go

```diff
@@ -831,6 +832,42 @@ var rpcClientTestCases = map[string][]rpcClientTestCase{
 			},
 		},
 	},
+	"getproof": {
```

Comment:
Could we add an error case for them as far?

Reply:
Will do.

Reply:
Done.